integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -470,41 +454,17 @@ def generate_map_with_empty_validity(spark, path):
             lambda spark, path: spark.read.parquet(path),
             data_path)
 
-@pytest.mark.parametrize('ts_write_data_gen', [('INT96', TimestampGen()),
-                                               ('TIMESTAMP_MICROS', TimestampGen(start=datetime(1, 1, 1, tzinfo=timezone.utc), end=datetime(1582, 1, 1, tzinfo=timezone.utc))),
```

Comment:
The timestamps here are setup to match the range that we know rebase is needed for. Are we sure that we are getting good enough coverage from just `test_datetime_roundtrip_with_legacy_rebase`?

Reply:
Good point. I'll change the range test.

Reply:
Done.